src/shared/ui/Message/MessagesWrapper.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { memo, useEffect } from 'react';
+import { Stack } from '@mui/material';
+import { getMessages } from 'entities/Task';
+import { Message } from './Message';
+import { useAppSelector } from '../../lib/hooks/redux';
+import { getPermission } from '../../lib/notification';
+
+import { MessageStyle as styles } from './Message.style';
+
+export const MessagesWrapper = memo(() => {
+    const messages = useAppSelector(getMessages);
+
+    useEffect(() => {
+        getPermission();
+    }, []);
+
+    if (messages.length) {
+        return (
+            <Stack sx={styles.wrapper} spacing={1}>
+                {messages.map((message) => <Message message={message} key={message.taskID} />)}
+            </Stack>
+        );
+    }
+
+    return null;
```

Comment:
it's better to move `return null;` under `if` statement istead of the main return value

Reply:
Fixed